keyboards/miiiw/blackio98/config.h

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2022 MIIIW <www.miiiw.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x3044
+#define PRODUCT_ID      0x98A1
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    MIIIW
+#define PRODUCT         MIIIW BlackIO 98
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 18
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 3
+
+/* Larger keys per scan */
+//#define QMK_KEYS_PER_SCAN 4
+
+/* force n-key rollover*/
+#define FORCE_NKRO
```

Comment:
This should be defined at a user level, as it can cause issues

Reply:
Fixed, have removed FORCE_NKRO macro.